src/Microsoft.Data/BaseColumn.cs

```diff
@@ -59,16 +59,25 @@ protected set
             get => GetValue(startIndex, length);
         }
 
+        /// <summary>
+        /// Called internally from Merge and GroupBy
```

Comment:
Merge does not exist yet, but it will soon

Reply:
That isn't a very good public description of what the method does. Can we give it a better summary?

---
In reply to: [301802932](https://github.com/dotnet/corefxlab/pull/2696#discussion_r301802932) [](ancestors = 301802932)